### DISCUSSION TITLE
Syntax of static methods

### DISCUSSION THREAD

**Discussion by evilpie** [mdn/mdn-community#361] [Content]
For static methods like [`createObjectURL`](https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL), MDN used to show the syntax like `URL.createObjectURL(object)` instead of `createObjectURL(object)`. I think the former is more useful in most cases. This makes it immediately obvious that this is a static method and the code can even be copied directly.

In this specific instance it's even a bit more problematic because there is no example code that shows how using this API would look like.

**Comment by teoli2003** [comment]
Hi, Tom!

You have a good point here.

For static methods, and only them, prefixing with the interface name would be:
1. correct
2. make it easier to copy and paste.

As we now have page types, this would be easy to fix automatically.

**Comment by OnkarRuikar** [comment]
- Fixed in https://github.com/mdn/content/pull/26134

**Reply by teoli2003** [reply]
Thanks, @OnkarRuikar. 
We still need to update this section https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Syntax_sections#constructors_and_methods

**Reply by OnkarRuikar** [reply]
> We still need to update this section [developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Syntax_sections#constructors_and_methods](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Syntax_sections#constructors_and_methods)

Done in https://github.com/mdn/content/pull/26139.

**Reply by teoli2003** [reply]
So it looks like this has been done! \o/ Thanks, @evilpie, for pointing this out.

**Reply by evilpie** [reply]
Thanks for fixing this so quickly!
